tokio/src/sync/mpsc/bounded.rs

```diff
@@ -16,6 +16,7 @@ use std::task::{Context, Poll};
 /// Send values to the associated `Receiver`.
 ///
 /// Instances are created by the [`channel`](channel) function.
+#[derive(PartialEq, Eq)]
 pub struct Sender<T> {
     chan: chan::Tx<T, Semaphore>,
 }
```

Comment:
A channel can only ever have one `Sender`, so this doesn't make much sense.

Reply:
For **Receiver**, yes it doesn't make much sense. For users it will be better to use false directly instead of depends on it's eq or same_channel method.